pyctbgui/services/ADC.py

```diff
@@ -6,8 +6,10 @@
 import pyqtgraph as pg
 from pyqtgraph import LegendItem
 
+from pyctbgui.utils import decoder
 from pyctbgui.utils.bit_utils import bit_is_set, manipulate_bit
 from pyctbgui.utils.defines import Defines
+from pyctbgui.utils.pixelmap import moench04_analog
```

Comment:
In case we have more pixel maps, maybe better import pixelmap as pm and then use it as pm.moench04_analog?